### PR TITLE
Switch to using Penny app to get a GH token

### PR DESCRIPTION
**Pull request by gwynne**
We no longer need to rely on a PAT held by the "Vapor Bot" user to run workflows that need permissions beyond those granted by default; we can use the Penny app installation to generate an access token.

### CODE REVIEW

File: Makefile
Comment (0xTim) [inline Makefile:1]: We need this as this is how installation is done on Linux

**Review by 0xTim — CHANGES_REQUESTED** [review]

File: Makefile
Comment (gwynne) [reply Makefile:1]: _Why..._ 😞 (Don't worry, I'm undeleting it.)

File: scripts/build.swift
Comment (0xTim) [inline scripts/build.swift:1]: I think this is used by the Makefile 😬

**Comment by 0xTim** [comment]
It's a pity #393 or #413 aren't ready!

File: scripts/build.swift
Comment (gwynne) [reply scripts/build.swift:1]: Already taken care of

**Review by 0xTim — APPROVED** [review]

**Pull request merged**